externs/browser/window.js

```diff
@@ -191,9 +191,9 @@ function setImmediate(callback, callbackParams) {}
 function setInterval(callback, opt_delay, callbackParams) {}
 
 /**
- * @param {Function|!TrustedScript|string} callback
+ * @param {function(...?):undefined|!TrustedScript|string} callback
```

Comment:
Investigated a bit more, and apparently there are places that pass in a nullable type here, which this patch forbids.

Example (inside base.js!):
javascript/closure/base.js:2927: ERROR - actual parameter 1 of global this.setTimeout does not match formal parameter
found   : (function(): undefined|null)
required: (TrustedScript|function(...?): undefined|string)
        goog.global.setTimeout(cb, 0);
                               ^^

Reply:
Hmm ... I had not considered that `Function` was nullable. Certainly the browser allows passing null. I will make it so by extracting another typedef and prefixing it with `?`. Hopefully that addresses this.